airbyte-cdk/python/airbyte_cdk/connector_builder/message_grouper.py

```diff
@@ -63,7 +66,9 @@ def get_message_groups(
                 if message_group.type == TraceType.ERROR:
                     error_message = f"{message_group.error.message} - {message_group.error.stack_trace}"
                     log_messages.append(LogMessage(**{"message": error_message, "level": "ERROR"}))
-
+            elif isinstance(message_group, AirbyteControlMessage):
+                if not latest_config_update or latest_config_update.emitted_at <= message_group.emitted_at:
```

Comment:
Nit: the fact that the control message we see here is related to config (and therefore that it's appropriate to assign it to the `latest_config_update` variable) is down in `_get_message_groups`; I wonder if it would be slightly clearer to move `and message.control.type == OrchestratorType.CONNECTOR_CONFIG` to this method and have `_get_message_groups` yield every control message.

Reply:
I see the point that it's more readable from the public `get_message_groups`. I honestly don't have a strong preference but if you think this is important enough, I can spend cycles and update it

Reply:
It's just a nit so no worries if you'd rather leave it as-is.